deny.toml

```diff
@@ -0,0 +1,66 @@
+[advisories]
+db-path = "~/.cargo/advisory-db"
+db-urls = ["https://github.com/rustsec/advisory-db"]
+vulnerability = "deny"
+unmaintained = "deny"
+yanked = "deny"
+notice = "deny"
+ignore = [
+    "RUSTSEC-2020-0016", # net2 deprecated - https://github.com/deprecrated/net2-rs/commit/3350e3819adf151709047e93f25583a5df681091
+    "RUSTSEC-2020-0056", # stdweb unmaintained - https://github.com/koute/stdweb/issues/403
+    "RUSTSEC-2021-0047", # security issue - https://github.com/gnzlbg/slice_deque/issues/90
+]
+
+[licenses]
+unlicensed = "deny"
+copyleft = "deny"
+allow = [
+    "MIT",
+    "Apache-2.0",
+    "BSD-3-Clause",
+    "ISC",
+    "Zlib",
+    "0BSD",
+    "BSD-2-Clause",
+    "CC0-1.0",
+    "MPL-2.0",
+]
+default = "deny"
+
+[[licenses.clarify]]
+name = "stretch"
+expression = "MIT"
+license-files = []
+
+[bans]
+multiple-versions = "deny"
+wildcards = "deny"
+highlight = "all"
+# Certain crates/versions that will be skipped when doing duplicate detection.
+skip = [
+    { name = "ahash", version = "=0.4.7" },
```

Comment:
I expect this section to regularly cause build breakage when minor versions are published.  Is there a way to make it less noisy, and if so, in what ways does it expose us?

Reply:
I don't think this will break often:
* for PRs it only runs if a `Cargo.toml` file has been modified, so not that often
* those are mostly older versions from libs that don't update their dependencies, so I don't think they will get updated with minor either

We can have `*` as the version, the only thing is that it won't warn us when we don't have a duplicate anymore or if we start depending on 3 versions of the crate (a triplicate?)

Reply:
Could you specify `0.4` instead of `=0.4.7`?

Reply:
oh sure, don't know why I went straight from exact version to wildcard. yup `0.4` works